miner/src/pool/queue.rs

```diff
@@ -303,6 +303,8 @@ impl TransactionQueue {
 		// Notify about imported transactions.
 		(self.pool.write().listener_mut().1).0.notify();
```

Comment:
This listener now seems obsolete, since it just duplicates the same data and is less efficient than the new channel-based one.

Reply:
@tomusdrw 

> Looks nice now, I'd like to see the other `Notifier` rewritten to futures as well, but we can leave that as a separate PR - please create an issue for this if that happens to be the case.

Ok, then let`s do it in other PR

>This listener now seems obsolete, since it just duplicates the same data and is less efficient than the new channel-based one.

Maybe we can make one Notifier by combining old one `Notifier` and `TransactionsPoolNotifier` and store two different lists of listeners for `parity_watchTransactionsPool` and `eth_pubsub` in it?